src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -117,7 +117,7 @@ public async Task HandleExtractRequest(ExtractParams parameters, RequestContext<
                     // Set connection details database name to ensure the connection string gets created correctly for DW(extract doesn't work if connection is to master)
                     connInfo.ConnectionDetails.DatabaseName = parameters.DatabaseName;
                     ExtractOperation operation = new ExtractOperation(parameters, connInfo);
-                    ExecuteOperation(operation, parameters, SR.ExtractDacpacTaskName, requestContext);
+                    ExecuteOperation(operation, parameters, SR.ExtractDacpacTaskName, requestContext, parameters.PackageFilePath);
```

Comment:
You're already passing in the whole parameters object so passing in package file path additionally seems unnecessary